api/backend/ecs/task_manager.go

```diff
@@ -171,13 +152,9 @@ func modelFromTasks(tasks []*ecs.Task) (*models.Task, error) {
 	}
 
 	model := &models.Task{
-		EnvironmentID: id.ClusterARNToECSEnvironmentID(*tasks[0].ClusterArn).L0EnvironmentID(),
-		PendingCount:  pendingCount,
-		RunningCount:  runningCount,
-		DesiredCount:  int64(len(tasks)),
-		TaskID:        id.ECSTaskID(*tasks[0].StartedBy).L0TaskID(),
-		Copies:        copies,
-		DeployID:      id.TaskDefinitionARNToECSDeployID(*tasks[0].TaskDefinitionArn).L0DeployID(),
+		RunningCount: runningCount,
+		PendingCount: pendingCount,
+		Copies:       copies,
```

Comment:
This seems to be add odds with how we removed `copies` as an argument, but for clarity, this is/was a slice of details about each copy of the task that was running. We still want to keep these details around, but since we only allow 1 copy of a task, this object is still a slice but will just always have a size of 1.

Reply:
👍